test/java/seedu/addressbook/commands/ViewCommandTest.java

```diff
@@ -0,0 +1,159 @@
+package seedu.addressbook.commands;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import static seedu.addressbook.util.TestUtil.assertCommandResult;
+
+public class ViewCommandTest {
+    private AddressBook addressBook;
```

Comment:
Hmph, we can't use `TestUtil.createAddressBook()` here because it throws a checked exception. However, if we step back a little, we see that `TestUtil.createAddressBook()` should *not* be passing on the `DuplicatePersonException` because it's *only used for test data*. (i.e. the same reason why `TypicalPersons` just `asserts false` if it encounters an `IllegalValueException`.)

So, I think you should just add this in to your PR:
```diff
diff --git a/test/java/seedu/addressbook/util/TestUtil.java b/test/java/seedu/addressbook/util/TestUtil.java
index 8f4eaf8..a1f0836 100644
--- a/test/java/seedu/addressbook/util/TestUtil.java
+++ b/test/java/seedu/addressbook/util/TestUtil.java
@@ -26,13 +26,16 @@ import seedu.addressbook.data.tag.UniqueTagList;
 public class TestUtil {
     /**
      * Creates an address book containing the given persons.
-     * @throws DuplicatePersonException if two or more given persons are the same
      */
-    public static AddressBook createAddressBook(Person... persons) throws DuplicatePersonException {
+    public static AddressBook createAddressBook(Person... persons) {
         AddressBook addressBook = new AddressBook();
 
         for (Person person : persons) {
-            addressBook.addPerson(person);
+            try {
+                addressBook.addPerson(person);
+            } catch (DuplicatePersonException e) {
+                throw new AssertionError(e);
+            }
         }
 
         return addressBook;
```

No point letting the quality of your PR suffer just because the code base just happens to be like this already.